main.js

```diff
@@ -0,0 +1,29 @@
+var alphabet = "abcdefghijklmnopqrstuvwxyz".split("");
+console.log(alphabet);
+var output = "";
+
+function stackLetters (array) {
+     for (var i = 0; i < array.length; i++) {
+     	output += array[i];
+     	console.log(output);
+	}
```

Comment:
Make sure you format your document before making a pull request.  Use a plugin to beautify your code.